indico/web/templates/indico_base.html

```diff
@@ -59,6 +59,7 @@
 </head>
 <body data-user-id="{{ session.user.id if session.user else 'null' }}"
       data-debug="{{ config.DEBUG|tojson|forceescape }}">
+    <ind-bypass-block-links></ind-bypass-block-links>
```

Comment:
WTF. Why can't custom elements be self-closing (https://stackoverflow.com/a/61698264/298479). This is ugly af. :/
Too bad we can't avoid it. Anyway, no problem, just slightly annoying since it's a bit ugly. The custom elements are still pretty cool even without this ability :)

Reply:
Yeah, it's a big wonky.